### ISSUE TITLE
Training VotNet with a custom dataset doesn't result in loss

### ISSUE THREAD

**Issue by boyagesmile** [open-mmlab/mmdetection3d#1903]
I trained VotNet with customed sun-rgbd dataset，and i get no result in loss
Here is the log
`2022-10-10 12:13:21,917 - mmdet - INFO - workflow: [('train', 1)], max: 100 epochs
2022-10-10 12:13:21,917 - mmdet - INFO - Checkpoints will be saved to /tmp/pycharm_project_652/work_dirs/votenet_16x8_sunrgbd-3d-custom-8class by HardDiskBackend.
/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /opt/conda/conda-bld/pytorch_1623448265233/work/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
/opt/conda/lib/python3.7/site-packages/torch/_tensor.py:575: UserWarning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (Triggered internally at  /opt/conda/conda-bld/pytorch_1623448265233/work/aten/src/ATen/native/BinaryOps.cpp:467.)
  return torch.floor_divide(self, other)
2022-10-10 12:13:58,812 - mmdet - INFO - Saving checkpoint at 1 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 2/2, 0.7 task/s, elapsed: 3s, ETA:     0s2022-10-10 12:14:01,865 - mmdet - INFO - 
+------------------+---------+---------+---------+---------+
| classes          | AP_0.25 | AR_0.25 | AP_0.50 | AR_0.50 |
+------------------+---------+---------+---------+---------+
| wire             | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
| pole             | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
| electricity_box  | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
| electricity_pole | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
| electricity_wire | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
+------------------+---------+---------+---------+---------+
| Overall          | 0.0000  | 0.0000  | 0.0000  | 0.0000  |
+------------------+---------+---------+---------+---------+
2022-10-10 12:14:01,866 - mmdet - INFO - Exp name: votenet_16x8_sunrgbd-3d-custom-8class.py
2022-10-10 12:14:01,866 - mmdet - INFO - Epoch(val) [1][2]	wire_AP_0.25: 0.0000, pole_AP_0.25: 0.0000, electricity_box_AP_0.25: 0.0000, electricity_pole_AP_0.25: 0.0000, electricity_wire_AP_0.25: 0.0000, mAP_0.25: 0.0000, wire_rec_0.25: 0.0000, pole_rec_0.25: 0.0000, electricity_box_rec_0.25: 0.0000, electricity_pole_rec_0.25: 0.0000, electricity_wire_rec_0.25: 0.0000, mAR_0.25: 0.0000, wire_AP_0.50: 0.0000, pole_AP_0.50: 0.0000, electricity_box_AP_0.50: 0.0000, electricity_pole_AP_0.50: 0.0000, electricity_wire_AP_0.50: 0.0000, mAP_0.50: 0.0000, wire_rec_0.50: 0.0000, pole_rec_0.50: 0.0000, electricity_box_rec_0.50: 0.0000, electricity_pole_rec_0.50: 0.0000, electricity_wire_rec_0.50: 0.0000, mAR_0.50: 0.0000
2022-10-10 12:14:41,339 - mmdet - INFO - Saving checkpoint at 2 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 2/2, 0.7 task/s, elapsed: 3s, ETA:     0s2022-10-10 12:14:44,322 - mmdet - INFO - `

**Comment by JingweiZhang12** [comment]
Your custom dataset is too small and you can set the log interval as a small value.
